src/cowrie/output/datadog.py

```diff
@@ -0,0 +1,63 @@
+"""
+Simple Datadog HTTP logger.
+"""
+
+from __future__ import annotations
+
+import json
+import platform
+
+from io import BytesIO
+from twisted.internet import reactor
+from twisted.internet.ssl import ClientContextFactory
+from twisted.web import client, http_headers
+from twisted.web.client import FileBodyProducer
+
+import cowrie.core.output
+from cowrie.core.config import CowrieConfig
+
+
+class Output(cowrie.core.output.Output):
+    def start(self):
```

Comment:
do we want to do some additional validation here? e.g. exit with an error if `api_key` isn't filled in

Reply:
Cowrie will handle all module errors for us. I'll just add a log message to warn if api_key is empty. However, I won't add the validation of the API key :)